src/firebase_functions/private/serving.py

```diff
@@ -67,10 +67,15 @@ def convert_value(obj):
 
 def functions_as_yaml(functions: dict) -> str:
     endpoints: dict[str, _manifest.ManifestEndpoint] = {}
+    required_apis: list[_manifest.ManifestRequiredApi] = []
     for name, function in functions.items():
         endpoint = function.__firebase_endpoint__
         endpoints[name] = endpoint
+        if hasattr(function, "__required_apis"):
+            for api in function.__required_apis:
+                required_apis.append(api)
```

Comment:
Are we doing something like:

https://github.com/firebase/firebase-functions/blob/master/src/runtime/loader.ts#L83

to merge required apis?